src/opticalEncoders-drift/opticalEncodersDrift.h

```diff
@@ -72,7 +72,7 @@ class OpticalEncodersDrift : public yarp::robottestingframework::TestCase {
 
     bool goHome();
     void setMode(int desired_mode);
-    void saveToFile(std::string filename, yarp::os::Bottle &b);
+    bool saveToFile(std::string filename, yarp::os::Bottle &b);
```

Comment:
```suggestion
    bool saveToFile(const std::string &filename, yarp::os::Bottle &b);
```
This is a better signature.